src/Command/InvalidatePathCommand.php

```diff
@@ -37,7 +37,7 @@ public function __construct(CacheManager $cacheManager = null)
     {
         if (2 <= func_num_args()) {
             @trigger_error('Passing a command name in the constructor is deprecated and will be removed in version 3', E_USER_DEPRECATED);
-            static::$defaultName = func_get_arg(1);
+            $this->setName(func_get_arg(1));
```

Comment:
i think you need to move this to after the parent::__construct now, otherwise we overwrite the custom name in the `configure` method. when setting the static name, the parent sees the new name, now it no longer sees it.